flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/JoinITCaseHelper.scala

```diff
@@ -34,7 +34,12 @@ object JoinITCaseHelper {
 
   def disableOtherJoinOpForJoin(tEnv: TableEnvironment, expected: JoinType): Unit = {
     val disabledOperators = expected match {
-      case BroadcastHashJoin => "NestedLoopJoin, SortMergeJoin"
+      case BroadcastHashJoin =>
+        // set up the broadcast join threshold to Long.MaxValue
+        // so that the threshold constraints are always met.
+        tEnv.getConfig.getConfiguration.setLong(
+          OptimizerConfigOptions.TABLE_OPTIMIZER_BROADCAST_JOIN_THRESHOLD, Long.MaxValue)
```

Comment:
If we remove this configuration, many cases fails because of "can not plan" exception.

Reply:
Thanks for the explanation